blackbox/docs/sql/information_schema.txt

```diff
@@ -340,6 +346,44 @@ Or get an overview of how many routines and routine types are available::
     +----------+--------------+
     SELECT 4 rows in set (... sec)
 
+Columns
+-------
+
++--------------------+-------------+
+| Name               | Data Type   |
++====================+=============+
+| routine_name       | String      |
++--------------------+-------------+
+| routine_type       | String      |
++--------------------+-------------+
+| routine_body       | String      |
++--------------------+-------------+
+| data_type          | String      |
++--------------------+-------------+
+| is_deterministic   | Boolean     |
++--------------------+-------------+
+| routine_definition | String      |
++--------------------+-------------+
+
+
+:routine_name:
+    Name of the routine (might be duplicated in case of overloading)
+:routine_type:
+    Type of the routine.
+    Can be ``FUNCTION``, ``ANALYZER``, ``CHAR_FILTER``, ``TOKEN_FILTER``
+    or ``TOKEN_FILTER``.
+:routine_body:
+    The language used for the routine implementation.
+    If it doesn't apply, then ``Null``.
+:data_type:
+    The return type of the function.
+    If it doesn't apply, then ``Null``.
+:is_deterministic:
+    If the routine is deterministic then `True`, else `False` (``Null`` if it doesn't apply).
+:routine_definition:
+    The signature of the user defined function (``Null`` if it doesn't apply).
```

Comment:
With signature I'd imply it's only the method signature, excluding the body - that's not the case, is it?